lib/razorpay/constants.rb

```diff
@@ -2,5 +2,5 @@
 module Razorpay
   BASE_URI = 'https://api.razorpay.com/v1/'.freeze
   TEST_URL = 'https://api.razorpay.com/'.freeze
-  VERSION = '1.2.0'.freeze
+  VERSION = '1.2.2'.freeze
```

Comment:
@akshaybhalotia Never bump version in a feature-PR. You don't want a situation where another PR bumps it as well. The release follows what we have here: https://github.com/razorpay/razorpay-php#release